game/ui/ui_mouse_icons/game.gd

```diff
@@ -55,10 +55,11 @@ func left_double_click_on_bg(position : Vector2) -> void:
 ## ITEM/HOTSPOT FOCUS ## 
 
 func element_focused(element_id : String) -> void:
-	var target_obj = escoria.esc_runner.get_object(element_id)
+	var target_obj = escoria.object_manager.get_object(element_id).node
 	$ui/tooltip_layer/tooltip.set_target(target_obj.tooltip_name)
 
-	if escoria.esc_runner.current_action != "use" && escoria.esc_runner.current_tool == null:
+	if escoria.action_manager.current_action != "use" \
```

Comment:
TODO: Direct mentions to a verb action will have to be removed because Escoria should not force a user to define a "use" verb action. There are defined in the verbs_menu where each button throws the action verb defined by the user.